src/components/TileCatalog/TileCatalog.jsx

```diff
@@ -128,8 +92,8 @@ const TileCatalog = ({
   const totalTiles = pagination && pagination.totalItems ? pagination.totalItems : 10;
 
   return (
-    <StyledContainerDiv className={className}>
-      <StyledCatalogHeader>
+    <div className={classNames(`${iotPrefix}--tile-catalog`, className)}>
```

Comment:
Swap the ordering of these 2 as consumers should be able to override our class if needed

Reply:
Makes sense. Will do.